### ISSUE TITLE
WordPress.com Hosted/Atomic & Masterbar: 404s for /masterbar/tracks-masterbar/tracks-events.min.js

### ISSUE THREAD

**Issue by JoshuaGoode** [Automattic/jetpack#17989]
Jetpack is generating a 404 on each WP-Admin page load for `wp-content/plugins/jetpack/_inc/build/masterbar/tracks-events.min.js?ver=9.2`

Discovered this while investigating issues related to https://github.com/Automattic/jetpack/pull/17939 and reviewing server logs. Confirmed it in the browser console across 5 Atomic sites.

#### Steps to reproduce the issue

1. Access the WP-Admin of any Jetpack site using the Masterbar module
2. Check browser console, should see 404s for `wp-content/plugins/jetpack/_inc/build/masterbar/tracks-events.min.js?ver=9.2`

#### What I expected
No 404s.

#### What happened instead
404s.

**Screenshots**
![image](https://user-images.githubusercontent.com/4887830/101262640-30a32480-370e-11eb-94b9-6bd128ae6b5a.png)

Internal discussion:
p1607032124088500-slack-C03AP0WBV

**Comment by eduardozulian** [comment]
Noted by a user in 26150644-hc

**Comment by jeherve** [comment]
Also reported in https://wordpress.org/support/topic/tracks-events-min-js-404/

I've updated the issue description accordingly.

**Comment by xpurichan** [comment]
Another report in 25732931-hc. This user found the problem because they would get a "There has been a critical error on your website. Please check your site admin email inbox for instructions." error if they tried to create a new post or access WP Admin.

They had code added to their My Custom Functions plugin which was causing this.

**Comment by jeherve** [comment]
> they would get a "There has been a critical error on your website.

This is most likely caused by another problem, fixed in #17939. 

In those scenarios, you can use the following code snippet to fix the issue:

```php
add_filter( 'jetpack_tools_to_include', function( $tools ) {
        $index = array_search(
                'calypsoify/class.jetpack-calypsoify.php',
                $tools
        );
        if ( $index ) {
                unset( $tools[$index] );
        }
        return $tools;
} );
```

**Comment by xpurichan** [comment]
Thanks much @jeherve ! Incidentally I re-enabled My Custom Functions on the user's website without adding the code snippet you provided and both WP Admin and post creation are working now. We'll keep that snippet noted in case the problem arises again for the customer.
